cli/src/config/templates.toml

```diff
@@ -459,14 +513,18 @@ separate(" ",
   format_short_commit_id(commit.commit_id()),
   if(commit.conflict(), label("conflict", "conflict")),
   if(config("ui.show-cryptographic-signatures").as_boolean(),
-    format_short_cryptographic_signature(commit.signature())),
+    format_short_cryptographic_signature(commit.signature())
+  ),
 )
-''' # TODO: Possibly add some kind of repo-unique salt to the hash methods, if it makes users feel more secure (right now, `main` will always be `branch-c8c7a` for every repo)
+'''
 
 'format_detailed_cryptographic_signature(signature)' = '''
 if(signature,
   separate(" ",
-    label("signature status " ++ signature.status(), signature.status()),
+    label(
+      separate(" ", "signature status", signature.status()),
```

Comment:
nit: just concatenate them. `signature.status()` will never be empty.

Reply:
Thanks, done.